test/dotnet-sln.Tests/GivenDotnetSlnList.cs

```diff
@@ -100,7 +100,8 @@ public void WhenInvalidSolutionIsPassedItPrintsErrorAndUsage(string solutionComm
                 .WithWorkingDirectory(projectDirectory)
                 .Execute(solutionCommand, "InvalidSolution.sln", "list");
             cmd.Should().Fail();
-            cmd.StdErr.Should().Be(string.Format(CommonLocalizableStrings.InvalidSolutionFormatString, "InvalidSolution.sln", LocalizableStrings.FileHeaderMissingError));
+            cmd.StdErr.Should().Contain(
+                String.Format(CommonLocalizableStrings.InvalidSolutionFormatString, Path.Combine(projectDirectory, "InvalidSolution.sln"), "").TrimEnd("."));
```

Comment:
Because the current error template has two parameters: the file name, and the error description, followed by a dot. 
We only want to see if it contains the first part (without the dot)

Reply:
Thanks, nice attention to detail. I don't have my computer but will fix by tomorrow morning :)